examples/HotelAds/AddHotelAd.cs

```diff
@@ -289,7 +289,10 @@ private static string AddCampaignBudget(GoogleAdsClient client, long customerId)
                 },
                 // Set the ad group.
                 AdGroup = adGroupResourceName,
-                Status = AdGroupAdStatus.Paused
+                // Set the ad group ad to enabled.  Setting this to paused will cause an error
```

Comment:
Knit: please reword to "Set the status of the ad group ad to enabled. Setting the status to paused causes an error for hotel campaigns. In order to pause a hotel campaign, set the status to paused at either the ad group or campaign level." (Note single spaces after periods.)